src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/IRemoteExtensionMethodImportCompletionService.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+#nullable enable
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.Completion.Providers
+{
+    internal interface IRemoteExtensionMethodImportCompletionService
+    {
+        Task<(IList<SerializableImportCompletionItem>, StatisticCounter)> GetUnimportedExtensionMethodsAsync(
+            DocumentId documentId,
+            int position,
+            string receiverTypeSymbolKeyData,
+            string[] namespaceInScope,
+            bool isExpandedCompletion,
```

Comment:
what is isExpandedCompletion for?  why do we need to pass that along.  is it basically just asking to "force" things or not?  if so, i would have it be "bool force" and doc that we normally time-gate things.  but 'forcing' ensures we will always compute the results.

Reply:
>  is it basically just asking to "force" things or not?
yes

Will change